src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx

```diff
@@ -187,7 +187,10 @@ class EstimateYourBenefitsForm extends React.Component {
 
   handleInternationalCheckboxFocus = e => {
     if (window.innerWidth <= SMALL_SCREEN_WIDTH) {
-      e.target.scrollIntoView();
+      const field = document.getElementById('classesOutsideUS');
+      if (field) {
```

Comment:
Is this a place for that optional chaining operator?

Reply:
doing `document.getElementById('classesOutsideUS')?.scrollIntoView();` gets `ESLint: Expected an assignment or function call and instead saw an expression.(no-unused-expressions)` error message when trying to commit